content/en/docs/setup/install/virtual-machine/index.md

```diff
@@ -0,0 +1,209 @@
+---
+title: Virtual Machine Installation
+description: Deploy istio and connect a virtual machine to it.
+weight: 40
+keywords:
+- kubernetes
+- virtual-machine
+- gateways
+- vms
+---
+
+Follow this guide to deploy Istio and connect a virtual machine to it.
+
+{{< warning >}}
+This guide relies heavily on the
+[stable plugin ca certificate](/docs/tasks/security/cert-management/plugin-ca-cert/) feature. Istio
+does not offer an industry standard recommendation for certificate management. Please consult your
+information security team when using any plugin certificate. The determination of your company
+policies for certificate management is the choice of your company.
+{{< /warning >}}
+
+{{< tip >}}
+This guide is tested and validated. The Istio authors feel this guide is suitable for experimentation
+but not production. Like all alpha features, this guide is subject to change.
+{{< /tip >}}
+
+## Prerequisites
+
+1. [Download the Istio release](/docs/setup/getting-started/#download)
+1. Perform any necessary [platform-specific setup](/docs/setup/platform-setup/)
+1. Check the requirements [for Pods and Services](/docs/ops/deployment/requirements/)
+1. Virtual machines must have IP connectivity to the ingress gateway in the connecting mesh, and optionally every pod in the mesh via L3 networking if enhanced performance is desired.
+
+## Prepare the guide environment
+
+1. Set the environment variables `"${ISTIO_DIR}"`, `"${WORK_DIR}"`, your cluster
+    name, and the service namespace. Ensure `"${WORK_DIR}"` is prefixed with `"${HOME}"`
+    (e.g. `WORK_DIR="${HOME}"/vmintegration`).
+
+    {{< text bash >}}
+    $ ISTIO_DIR="<the directory containing an unarchived version of Istio>"
+    $ CLUSTER_NAME="<the name of your cluster>"
+    $ SERVICE_NAMESPACE="<the name of your service namespace>"
+    $ WORK_DIR="<a certificate working directory>"
+    {{< /text >}}
+
+1. Create the `"${WORK_DIR}"/"${CLUSTER_NAME}"/"${SERVICE_NAMESPACE}"` working directories.
+
+    {{< text bash >}}
+    $ mkdir -p "${WORK_DIR}"/"${CLUSTER_NAME}"/${SERVICE_NAMESPACE}"
+    {{< /text >}}
+
+## Install the Istio control plane
+
+1. Execute the following commands to create certificates and install them in your
```

Comment:
I'd like to explain to the user rather than just tell them to run a command, can we reword to explain what this is doing? This command does a LOT of work and it isn't obvious what parts the user can choose not to do.

Reply:
This is a hard to describe because much magic is done in the makefile. @myidpt can you offer appropriate wording here?